themes/default/content/docs/intro/pulumi-service/ci-cd-integration-assistant.md

```diff
@@ -17,46 +17,31 @@ to [Organizations]({{< relref "/docs/intro/pulumi-service/organizations" >}}), n
 
 <!--more-->
 
-A version control system (VCS) provides team collaboration capabilities and ensures that the source code for your Pulumi project is not
-just on a single developer's machine. Using a CI/CD system makes your team more productive, by automatically deploying your Pulumi stacks.
-So that code is delivered to production -- meaning less of a lag between commits and what your end users are seeing.
-
-The assistant contains a guided experience to help teams configure a VCS as well as automate it with a
+A version control system (VCS) provides team collaboration capabilities and ensures that the source code for your Pulumi project is not on a single developer's machine. Using a CI/CD system makes your team more productive, by automatically deploying your Pulumi stacks. The assistant contains a guided experience to help teams configure a VCS as well as automate it with a
```

Comment:
nit: A helpful thing with markdown is to keep a sentence on its own line, that way for reviews, you get the diff of just that line and not a huge change.